### ISSUE TITLE
Rate of progress tracker

### ISSUE THREAD

**Issue by Tapha** [Tanibox/tania#31]
- A feature that will allow the user to track the rate of progress on a target by day, week, month, year.

- Add the factors determining the rate of progress.

- Make automatic calculations that project where you will be in relation to the target based on the factors inputted.

**Comment by Tapha** [comment]
This would likely live within the 'farm' or 'area' sections. Since we want to adopt the 'belongs to' philosophy.

**Comment by bepitulaz** [comment]
From my understanding, what you mean about **target** is tasks to do for the employee, isn't? Something like this:

Let says you got an order from the client to grow tomatoes. Then you set it as "The Tomatoes Project" with a due date 4 months from now. So, you want to track your employee for this project. When they do the seeding; when they should add the fertiliser; when they should harvest, and so on.

Is it correct?

**Comment by Tapha** [comment]
Yes, basically. A task would be a subset of a target, and an employee would be an owner of a task (or multiple tasks). A series of tasks accomplishes a target. The progress would be tracked by tasks set vs tasks completed.

**Comment by bepitulaz** [comment]
Ok, got it! 👍
